lib/julia/builder.rb

```diff
@@ -9,13 +9,25 @@ def initialize(collection, csv_options = Hash.new)
     end
 
     def self.column(keyname, action = nil, &block)
-      self.columns[keyname] = Action.new(keyname, action, &block)
+      if !self.model_class.nil? && self.model_class.respond_to?(:human_attribute_name)
+        translated_key = self.model_class&.human_attribute_name(keyname)
+      end
+
+      self.columns[translated_key || keyname] = Action.new(keyname, action, &block)
```

Comment:
correct syntax:
```ruby
def self.column(keyname, action = nil, &block)
  if model_class.respond_to?(:human_attribute_name)
    translated_key = model_class.human_attribute_name(keyname)
  end
 
  columns[translated_key || keyname] = Action.new(keyname, action, &block)
end   
```

Reply:
I also think `model_class` should be memoized since it's being executed twice for each column